Package.swift

```diff
@@ -22,6 +22,14 @@ let package = Package(
         )
     ],
     targets: [
+        .target(
```

Comment:
If we possibly could, can we make this target static? Same with the other targets besides `KombuchaCLI` and the test targets

Reply:
Yes, I would prefer fix this in a separate PR.